drivers/ioexpander/gpio.c

```diff
@@ -511,20 +511,14 @@ static int gpio_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
  * Name: gpio_pin_register
  *
  * Description:
- *   Register GPIO pin device driver.
- *
- *   - Input pin types will be registered at /dev/gpinN
- *   - Output pin types will be registered at /dev/gpoutN
- *   - Interrupt pin types will be registered at /dev/gpintN
- *
- *   Where N is the provided minor number in the range of 0-99.
+ *   Register GPIO pin device driver at /dev/gpioN, where N is the provided
+ *   minor number in the range of 0-99.
```

Comment:
Didn't some previous commit remove the limit of 99? Comment still valid?

Reply:
I was not aware. I will update the comments. So there's no limit anymore? (except for the maximum number that the int type can hold, of course).

Reply:
Here is the PR: https://github.com/apache/incubator-nuttx/pull/4697. Sorry, forget to update the comment.